<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="18210" started-at="2020-10-14T10:36:43Z" finished-at="2020-10-14T10:37:01Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="18210" started-at="2020-10-14T10:36:43Z" finished-at="2020-10-14T10:37:01Z">
      <class name="com.test.negative">
        <test-method status="PASS" signature="invokeBrowser()[pri:0, instance:com.test.negative@29fc1a2b]" name="invokeBrowser" is-config="true" duration-ms="11302" started-at="2020-10-14T10:36:43Z" finished-at="2020-10-14T10:36:54Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invokeBrowser -->
        <test-method status="FAIL" signature="signIn(java.lang.String, java.lang.String)[pri:0, instance:com.test.negative@29fc1a2b]" name="signIn" duration-ms="4214" started-at="2020-10-14T10:36:55Z" data-provider="getData" finished-at="2020-10-14T10:36:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[akashrinki@yahoo.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[aksjksdj]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[The webpage is not Correct expected [My account - My Store] but found [Login - My Store]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: The webpage is not Correct expected [My account - My Store] but found [Login - My Store]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.test.negative.signIn(negative.java:57)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
at java.lang.reflect.Method.invoke(Unknown Source)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- signIn -->
        <test-method status="PASS" signature="flush(org.testng.ITestResult)[pri:0, instance:com.test.negative@29fc1a2b]" name="flush" is-config="true" duration-ms="584" started-at="2020-10-14T10:36:59Z" finished-at="2020-10-14T10:37:00Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=signIn status=FAILURE method=negative.signIn(java.lang.String, java.lang.String)[pri:0, instance:com.test.negative@29fc1a2b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flush -->
        <test-method status="PASS" signature="closing()[pri:0, instance:com.test.negative@29fc1a2b]" name="closing" is-config="true" duration-ms="1063" started-at="2020-10-14T10:37:00Z" finished-at="2020-10-14T10:37:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closing -->
      </class> <!-- com.test.negative -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
